content/quickstart/java/metrics.md

```diff
@@ -145,24 +158,40 @@ public class Repl {
     private static String processLine(String line) {
         return line.toUpperCase();
     }
+
+    private static void readEvaluateProcessLine(BufferedReader in) throws IOException {
+        System.out.print("> ");
+        System.out.flush();
+        String line = in.readLine();
+        String processed = processLine(line);
+        System.out.println("< " + processed + "\n");
+    }
 }
 ```
 
-Install required dependencies:
+Return to the project's root directory and install required dependencies:
+
 ```bash
+cd ../../../../../../
```

Comment:
Actually I'd rather we don't do this, if a user changes directories spuriously, they'll complain that they can't find the root following exactly these commands. I think we could avoid having to direct them with these instructions by:
1) Not even running the archetype code, it's unnecessary. They can just paste `pom.xml` but also `mkdir -p src/main/java/io/opencensus/quickstart` 
2) Paste `Repl.java` into `src/main/java/io/opencensus/quickstart/Repl.java`

Reply:
However, I can send a follow up PR to use the suggestions I've made since that might be out of scope for this PR.

Reply:
I'll push a new commit that incorporates these changes, thank you!